samples/MvcSample.Web/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "commands": {
-    "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
+    "web": "Microsoft.AspNet.Server.Kestrel",
```

Comment:
This project is in the midst of a fiery death (see #3574). Don't bother with any changes here.